lib/client/debug/debug.go

```diff
@@ -55,8 +56,9 @@ func NewClient(socketPath string) *Client {
 		clt: &http.Client{
```

Comment:
While you're here, could you also add a `CheckRedirect` that errors unconditionally?

Reply:
Totally, done